auth/user.go

```diff
@@ -29,3 +32,31 @@ func UpdateUser(db *gorm.DB, user *User) (bool, error) {
 
 	return result.RowsAffected == 1, result.Error
 }
+
+// create JWT Token
+func CreateToken(user *User) (string, error) {
+	token := jwt.NewWithClaims(tokenSigningMethod(os.Getenv("JWT_ALGO")), jwt.MapClaims{
+		"username":       user.Username,
+		"name":           user.Name,
+		"email":          user.Email,
+		"githubusername": user.GithubUsername,
+		"githubtoken":    user.GithubToken,
```

Comment:
I would take out the github fields here, especially the token. We'll want to keep that private